solar-land/scss/_typography.scss

```diff
@@ -0,0 +1,34 @@
+.quicksand-light {
+  font-family: $primaryFontFamily, sans-serif;
+  font-weight: 300;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
+
+.quicksand-regular {
+  font-family: $primaryFontFamily, sans-serif;
+  font-weight: 400;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
+
+.quicksand-medium {
+  font-family: $primaryFontFamily, sans-serif;
+  font-weight: 500;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
+
+.quicksand-semibold {
+  font-family: $primaryFontFamily, sans-serif;
+  font-weight: 600;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
+
+.quicksand-bold {
+  font-family: $primaryFontFamily, sans-serif;
+  font-weight: 700;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
```

Comment:
These 4 classes have 3 same declarations instead of repeating these declarations in all of them it would be less code to write them like this : 
```
.quicksand-light,
.quicksand-regular,
.quicksand-medium ,
.quicksand-semibold,
.quicksand-bold
{
  font-family: $primaryFontFamily, sans-serif;
  font-optical-sizing: auto;
  font-style: normal;
}

.quicksand-light {
  font-weight: 300;
}

.quicksand-regular {
  font-weight: 400;
}

.quicksand-medium {
  font-weight: 500;
}

.quicksand-semibold {
  font-weight: 600;
}

.quicksand-bold {
  font-weight: 700;
}
```

Reply:
And also it could be a loop as well :)

Reply:
Oh, thanks. Two practical suggestions that I should have paid attention to during the coding phase.